WalletWasabi.Fluent/Views/Wallets/WalletView.axaml

```diff
@@ -24,7 +24,6 @@
               <Button.IsVisible>
                 <MultiBinding Converter="{x:Static BoolConverters.And}">
                   <Binding Path="PreferPsbtWorkflow" />
-                  <Binding Path="IsWalletBalanceZero" Converter="{x:Static BoolConverters.Not}" />
```

Comment:
Broadcast button should not be visible when the wallet is empty.

Reply:
Where can I see this broadcast button?

Reply:
When you use Wasabi with an HW wallet. In the wallet settings, there is an option for `PSBT workflow`, when you have it enabled the broadcast button will appear where the send and receive are. And also the sending procedure changes a bit, there will be a button for Save PSBT at the end.

This workflow is like... the user creates a psbt, move it to the hw wallet, sign it. And then broadcast with Wasabi. That is why the broadcast button appears.

Reply:
> Broadcast button should not be visible when the wallet is empty.

Done. AFAIK even if it was visible it won't cause problems because it just broadcasts any transaction.

Should I also hide the `PSBT workflow` when the wallet is empty? Because when it is empty `PSBT workflow` does nothing. Maybe this is not necessary but I am just pointing that out.